MalcolmMcFly/RepoRadar-Android
# Added documentation

## Describe your changes
- [x] Added pull request template
- [x] Added dokka
- [x] Updated gitignore

## How should this be tested?
Verify that docs are successfully generating by running `./gradlew dokkaHtml` from the root level of the project. The docs can be opened from a web browser locally from `app/build/dokka/html/index.html`.
